pilot/pkg/networking/core/v1alpha3/envoyfilter/listener_patch.go

```diff
@@ -468,6 +468,11 @@ func doHTTPFilterListOperation(patchContext networking.EnvoyFilter_PatchContext,
 				continue
 			}
 
+			// In case of INSERT_FIRST, if a match is found, still insert it at the top of the filterchain.
```

Comment:
For insert_first why do we need to find a position?

Reply:
The goal is to have EnvoyFilter_Patch_INSERT_BEFORE and EnvoyFilter_Patch_INSERT_FIRST handled together.